openedx/core/djangoapps/user_authn/views/tests/test_register.py

```diff
@@ -393,7 +392,7 @@ class RegistrationViewTest(ThirdPartyAuthTestMixin, UserAPITestCase):
 
     def setUp(self):  # pylint: disable=arguments-differ
         super(RegistrationViewTest, self).setUp()
-        self.url = reverse("user_api_registration")
```

Comment:
Don't override the exisitng `self.url`. Keep it for testing as is. You should have separate tests for v2.